Description: Fix the path of autorunner.sh in the usb-discovery script 
 .
 usb-autorunner (0.1-1) unstable; urgency=medium
 .
   * Initial release
Author: Zoheir Chine <chine.zoheir@gmail.com>

--- usb-autorunner-0.1.orig/usb-discovery.sh
+++ usb-autorunner-0.1/usb-discovery.sh
@@ -19,7 +19,7 @@ do
 			if [[ ! $(lsblk /dev/$device -o LABEL -n) =~ $filteredDeviceLabel ]]
 			then
 				echo "⚙️  Trigger USB Autorunner on '/mnt/usb/$device'..."
-				{{ usb_autorunner_install_path }}/autorunner.sh $(lsblk /dev/$device -o MOUNTPOINT -n)
+				/opt/usb-autorunner/autorunner.sh $(lsblk /dev/$device -o MOUNTPOINT -n)
 			fi
 			echo "⚙️  Unmounting device..."
 			umount /dev/$device
